sdk/dotnet/APIGateway/V1/Api.cs

```diff
@@ -93,6 +93,12 @@ private static CustomResourceOptions MakeResourceOptions(CustomResourceOptions?
             var defaultOptions = new CustomResourceOptions
             {
                 Version = Utilities.Version,
+                ReplaceOnChanges =
+                {
+                    "apiId",
```

Comment:
It looks like you're generating a `ReplaceOnChanges` for every property, correct? If so, I think it would be clearer to use the wildcard `*` instead.

Reply:
Its not for every property here - just the required query and all path properties. I have a separate change I'd like to add where we mark immutable resources (resources without an update endpoint) to require replace on change. In that case every property would indeed be marked replace on change. Unfortunately the schema as it stands doesn't support specifying a wildcard explicitly but something we can consider adding.